package.json

```diff
@@ -13,12 +13,10 @@
     "tar"
   ],
   "devDependencies": {
-    "node-gyp": "3.x",
     "tap": "7.x"
   },
   "scripts": {
-    "test": "tap --reporter tap tests/api_tests.js",
-    "install": "node-gyp rebuild"
```

Comment:
At the moment our build scripts rely on this "install" line being here and do a simple search and replace when building the npm package. I'd prefer to leave this line in, I know node-gyp rebuild is the default but I don't think it causes any problems to be explicit and it would save us having to rework the build scripts to do something more complicated.